### DISCUSSION TITLE
NetPanzer Weekly AppImage (Unstable)

### DISCUSSION THREAD

**Discussion by andy5995** [netpanzer/netpanzer#82] [Announcements]
## Important Note

Due to issue #71 (which we hope to fix soon), the AppImage can't be executed normally (you'll get a segfault). But it's still simple to try:

    `chmod +x` the AppImage
    Run `NetPanzer-[appimage-name].AppImage --appimage-mount`

In another terminal, cd to '/tmp'. Check for a directory named something similar to '.mount_NetPanin3MdD/'

Change to that, then to 'usr/bin' (NOT '/usr/bin'), and then run `./netpanzer`

Thank you for your patience.

<hr /><em>This discussion was created from the release <a href='https://github.com/netpanzer/netpanzer/releases/tag/appimage-weekly'>NetPanzer Weekly AppImage (Unstable)</a>.</em>

**Comment by andy5995** [comment]
@winrid  Why we use Ubuntu focal docker containers for building the AppImage. https://github.com/orgs/AppImage/discussions/1254#discussioncomment-5312266

**Comment by andy5995** [comment]
#71 has been fixed so now the AppImage runs from anywhere. (you can ignore the info written by OP)

**Comment by andy5995** [comment]
Now we have a weekly arm64 (aarch64) AppImage available https://github.com/netpanzer/netpanzer/releases/tag/appimage-weekly

**Comment by andy5995** [comment]
@winrid For the 0.9.0 release, you can download the last weeky appimages, rename them, and upload them to the release.

**Reply by winrid** [reply]
@andy5995 done. I tried earlier but I couldn't find them as I was looking under Actions runs.
